tools/c7n_azure/c7n_azure/actions.py

```diff
@@ -345,3 +342,66 @@ def process_resource(self, resource):
             return
 
         self.untag_action({'tags': candidates}, self.manager).process([resource])
+
+
+class Notify(BaseNotify):
+
+    batch_size = 50
```

Comment:
14 bytes may be more wiggle room than we need? Hate to give up 21% of the message size.  Looks like they left a 6-byte buffer in the AWS implementation which makes me think 58 would work fine at least?

Reply:
Batch size is actually not in KB, but in the number of resources. It’s a bit hard to predict how many resources we can batch together as resources differ, but 50 seems safe enough.